addon/components/influencer/profile-description.hbs

```diff
@@ -1,103 +0,0 @@
-<img src={{profile.avatarUrl}} class="img-preloader" onload={{action "imageHasLoaded"}}/>
-<div class='profile-description__avatar'>
-  {{#if isSelectable}}
-    {{#if (eq overlayType 'selection')}}
-      {{#toggle-area value=_selected callbackAction=toggleAction icon=selectedIcon}}
-        <div class="profile-description__bgimg" style={{bgImgStyle}}></div>
-      {{/toggle-area}}
```

Comment:
You won't find this in https://github.com/upfluence/facade-web/pull/440 as it's a behaviour we don't have anymore. When used in Inbox, this was used to select/unselect an influencer (while the split-overlay-area is used for the select/reject feature that's in Facade only)